README.md

```diff
@@ -25,7 +25,9 @@ states and produce helpful logs.
 
 Install and run Matrix by doing the following:
 
-    sudo pip3 install 'git+https://github.com/juju-solutions/matrix.git'
+    git clone https://github.com/juju-solutions/matrix.git
+    cd matrix
+    sudo pip3 install . -f wheelhouse
```

Comment:
There are instructions for installing in a venv further down.  The point of the default install instructions *is* to install at the system (or container) level.

A snap for this would be good, but it would have to be unconfined.  It's worth looking in to, though, so I created #48